packages/grafana-schema/src/schema/mudball.cue

```diff
@@ -249,12 +253,19 @@ VizLegendOptions: {
 // TODO docs
 BarGaugeDisplayMode: "basic" | "lcd" | "gradient" @cuetsy(kind="enum")
 
+// TODO docs
+TableCellOptions: {
+	displayMode: TableCellDisplayMode | *"auto"
+	gaugeDisplayMode?: BarGaugeDisplayMode
+	backgroundDisplayMode?: BackgroundDisplayMode
+} @cuetsy(kind="interface")
+
 // TODO docs
 TableFieldOptions: {
 	width?:      number
 	minWidth?:   number
 	align: FieldTextAlignment | *"auto"
-	displayMode: TableCellDisplayMode | *"auto"
```

Comment:
you are potentially breaking community plugins when changing this and other properties in schema and ui (see the levitate report). The recommended way to do this is to deprecate the properties and remove them in the next major version.

Reply:
Thanks for the feedback! This is still very much in progress and will make sure to mark a deprecated and opposed to removing them as I clean this up 😄